client/src/connection/itc/index.ts

```diff
@@ -421,7 +411,12 @@ export class ITCSession extends Session {
    * @param data a buffer of stdout output from the child process.
    */
   private onShellStdOut = (data: Buffer): void => {
-    const output = data.toString();
```

Comment:
It was intended to NOT trimEnd. At line 423, it will ignore empty lines. If it was trimed, empty lines will never be displayed. But we do need some empty lines to make the log look pretty.

Reply:
I don't remember adding trimEnd there. I've removed it in https://github.com/sassoftware/vscode-sas-extension/pull/880/commits/cfbd34ab90a037e8df31905535cfb89d8e0d886e and will make sure to re-test